.changeset/thirty-ravens-fly.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
```

Comment:
I think we also need to add this for `@astrojs/node` and `@astrojs/telemetry` so they all have new versions published.

Reply:
`@astrojs/telemetry` is a good catch, node and vercel shouldn't need a bump though. I think it would apply retroactively to them.